kernel/data-pipeline/dialect/opengauss/src/main/java/org/apache/shardingsphere/data/pipeline/opengauss/ingest/OpenGaussWALDumper.java

```diff
@@ -96,7 +94,7 @@ protected void runBlocking() {
                 int times = reconnectTimes.incrementAndGet();
                 log.error("Connect failed, reconnect times={}", times, ex);
                 if (isRunning()) {
-                    Thread.sleep(5000L);
+                    sleepIgnoreInterrupt(5000L);
```

Comment:
When InterruptedException is thrown, `break` while loop might be necessary, but not just ignore it. It might cause dumper could not be paused.